pandas/tests/series/indexing/test_indexing.py

```diff
@@ -768,3 +768,20 @@ def test_head_tail(test_data):
     assert_series_equal(test_data.series.head(0), test_data.series[0:0])
     assert_series_equal(test_data.series.tail(), test_data.series[-5:])
     assert_series_equal(test_data.series.tail(0), test_data.series[0:0])
+
+
+def test_reset_index_drop_errmsg():
+    # https://github.com/pandas-dev/pandas/issues/20925
+    # Data for Test Case 1 and 2
+    s = pd.Series(range(4))
+    # Test Case 1
```

Comment:
put blank lines between cases. Don't use a 'Test Case 1' as a lable, either remove of put what it is testing